Common/HistValidation.py

```diff
@@ -0,0 +1,61 @@
+import ROOT
+#import statsmodels.stats.proportion as ssp
+import os
+ROOT.gROOT.ProcessLine(".include "+ os.environ['ANALYSIS_PATH'])
+hist_file = ROOT.TFile("Corrections/data/BTV/2018_UL/btagEff.root","READ") # take as argument
+
+def GetKeyNames( self, dir = "" ):
+        self.cd(dir)
+        return [key.GetName() for key in ROOT.gDirectory.GetListOfKeys()]
+ROOT.TFile.GetKeyNames = GetKeyNames
+keyList = hist_file.GetKeyNames()
+#$print( "\nKeys in file:", keyList)
+nums = ["jet_pt_eta_0_Loose","jet_pt_eta_0_Medium","jet_pt_eta_0_Tight","jet_pt_eta_4_Loose","jet_pt_eta_4_Medium","jet_pt_eta_4_Tight","jet_pt_eta_5_Loose","jet_pt_eta_5_Medium","jet_pt_eta_5_Tight",]
+denums = ["jet_pt_eta_0", "jet_pt_eta_4", "jet_pt_eta_5"]
+i=0
+for key in keyList:
+    if i>1: break
+    hist = hist_file.Get(key)
+    #print(f"""for key {key} the histogram has {hist.GetNbinsX()*hist.GetNbinsY()} bins
+                  #""")
+    for x_bin_number in range(1, hist.GetNbinsX()-1):
+        for y_bin_number in range(1, hist.GetNbinsY()-1):
+            bin_number = hist.GetBin(x_bin_number,y_bin_number)
+            bin_content = hist.GetBinContent(bin_number)
+            if(bin_content==0):
+                print(f""" key = {key},
+                    bin_number = {bin_number}, bin_content = {bin_content},
+                    bin_center_X = {hist.GetXaxis().GetBinCenter(x_bin_number)},
+                    bin_center_y = {hist.GetYaxis().GetBinCenter(x_bin_number)}""")
```

Comment:
why this is needed?

Reply:
for personal check

Reply:
I mean, we need to print when the bin content is 0!!

Reply:
but we have loop over all bins below. why to have `for key in keyList` and `for jet_fl in jet_flavors`?

Reply:
Because we need to discriminate numerator from denumerator